ghettosync/role_slave.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/sh
+
+set -e
+
+abort() {
+    echo "status_standby aborted: $@"
+    exit 1
+}
+
+log() {
+    echo "$(date --iso-8601=seconds) role_slave: $@" >> /var/log/ghettosync/ghettosync.log
+}
+
+ROLE_FILE="ghettosync_role"
+ROLE_PATH="/var/www/html/$ROLE_FILE"
+
+TRIGGER_FILE="/tmp/postgresql.trigger.5432"
+
+VAR_MAIN="/var/lib/postgresql/10/main"
+ETC_MAIN="/etc/postgresql/10/main"
+
+REPUSER_PASSWORD="postgres_ec0nomy"
```

Comment:
Can we read secrets like this from .env so that they're not in the public repo? Can use something like
```
[[ -f .env ]] && export $(cat .env | xargs)
```
(that's how start.sh does it).

Things like PARTNER_IP would also be nice so that we don't have to modify the code on each node.